examples/jsm/renderers/webgpu/WebGPUBackground.js

```diff
@@ -41,6 +46,38 @@ class WebGPUBackground {
 			_clearAlpha = 1;
 			forceClear = true;
 
+		} else if ( background.isNode === true ) {
```

Comment:
Can we actually create `BackgroundNode`, which will do this (at least creating a mesh and aligning it to the camera) by itself? I think it will allow to more easily implement in the future the `Background` suggestion.

Reply:
That sounds interesting...